source/config/configSpec.py

```diff
@@ -225,6 +225,9 @@
 	# 0:default, 1:Only when necessary, 2:yes, 3:no
 	allowInChromium = integer(0, 3, default=0)
 
+[annotations]
+	reportDetails = boolean(default=false)

```

Comment:
For some experimental features I advise using an integer, so that we can have 3 way selections: (new behavior, old behavior, and default). This allows us to differentiate between those who have explicitly turned off a feature, and those who have just left it as the default. I think this situation is more important if we expect the flag to be long lived, and the default behavior to change. 

For annotations I think a boolean is fine, since I hope we are able to remove the option once we are happy with the behavior. A bool allows us to turn it on to test, but most users won't be bothered by a half implementation.

Just to illustrate my first point. Imagine we want to do the following:
- Add a new feature, default to "off", behind a flag for dev purposes.
- Later when the implementation is complete, it stays configurable, but default to "on". We want all users to give it a try, but we are aware that it won't work for everyone.

Meanwhile:
- User A, aware of the early development, decides that feature is not for them. Chooses to disable it. We don't want to change their explicitly chosen preference.
- User B, not aware of the feature. They should try the feature when the default is changed to "on".